test/cypress/integration/card/moto-mask-payment.spec.js

```diff
@@ -0,0 +1,195 @@
+const cardPaymentStubs = require('../../utils/card-payment-stubs')
+const gatewayAccountId = 42
+
+describe('Standard card payment flow', () => {
+  const tokenId = 'be88a908-3b99-4254-9807-c855d53f6b2b'
+  const chargeId = 'ub8de8r5mh4pb49rgm1ismaqfv'
+
+  const validPayment = {
+    cardNumber: '4444333322221111',
+    expiryMonth: '01',
+    expiryYear: '30',
+    name: 'Valid Paying Name',
+    securityCode: '012',
+    addressLine1: '10 Valid Paying Address',
+    city: 'London',
+    postcode: 'E1 8QS',
+    email: 'validpayingemail@example.com'
+  }
+
+  const createPaymentChargeStubsNoMoto = cardPaymentStubs.buildCreatePaymentChargeStubs(
+    tokenId, chargeId, 'en', gatewayAccountId, {}, {}, {
+      allowMoto: false
+    })
+
+  const createPaymentChargeStubsMotoMaskCardNumberOnly = cardPaymentStubs.buildCreatePaymentChargeStubs(
+    tokenId, chargeId, 'en', gatewayAccountId, {}, {}, {
+      allowMoto: true,
+      motoMaskCardNumberInput: true
+    })
+
+  const createPaymentChargeStubsMotoMaskSecurityCodeOnly = cardPaymentStubs.buildCreatePaymentChargeStubs(
+    tokenId, chargeId, 'en', gatewayAccountId, {}, {}, {
+      allowMoto: true,
+      motoMaskCardSecurityCodeInput: true
+    })
+
+  const createPaymentChargeStubsMotoMaskCardNumberAndSecurityCode = cardPaymentStubs.buildCreatePaymentChargeStubs(
+    tokenId, chargeId, 'en', gatewayAccountId, {}, {}, {
+      allowMoto: true,
+      motoMaskCardNumberInput: true,
+      motoMaskCardSecurityCodeInput: true
+    })
+
+  const checkCardDetailsStubs = cardPaymentStubs.checkCardDetailsStubs(chargeId)
+
+  const confirmPaymentDetailsStubs = cardPaymentStubs.confirmPaymentDetailsStubs(chargeId, validPayment)
+
+  beforeEach(() => {
+    // this test is for the full process, the session should be maintained
+    // as it would for an actual payment flow
+    Cypress.Cookies.preserveOnce('frontend_state')
+  })
+
+  describe('Secure card payment page - MOTO NOT enabled', () => {
+    it('Should setup the payment and load the page', () => {
+      cy.task('setupStubs', createPaymentChargeStubsNoMoto)
+      cy.visit(`/secure/${tokenId}`)
+
+      cy.location('pathname').should('eq', `/card_details/${chargeId}`)
+      cy.window().its('chargeId').should('eq', `${chargeId}`)
+    })
+
+    it('Card number & security code input elements should NOT mask digits', () => {
+      cy.get('#card-no').invoke('attr', 'type').should('contain', 'text')
+      cy.get('#cvc').invoke('attr', 'type').should('contain', 'text')
```

Comment:
I think we can assert on the whole value here: `should('eq', 'text')`

Reply:
Good spot - done